actionpack/lib/action_dispatch/middleware/templates/rescues/_trace.html.erb

```diff
@@ -1,52 +1,54 @@
-<% names = @traces.keys %>
+<% names = traces.keys %>
```

Comment:
With this we could spare the multiple instances for `error_index: 0` in the render calls.

```ruby
<% error_index = local_assigns.fetch(:error_index, 0) %>
```